org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/alloy4/TableView.java

```diff
@@ -152,7 +154,19 @@ public static String revertSuffix(String cmd) {
             TupleSet instanceTuples = instance.tuples(s.label);
             if (instanceTuples != null) {
 
-                SimTupleset sigInstances = toSimTupleset(instanceTuples);
+                List<SimTuple> instancesArray = toArrayList(instanceTuples);
+                Collections.sort(instancesArray, new Comparator<SimTuple>() {
+                        @Override
+                        public int compare(SimTuple simTuple1, SimTuple simTuple2) {
+                            String[] coll1 = simTuple1.get(0).toString().split("\\$");
+                            String[] coll2 = simTuple2.get(0).toString().split("\\$");
+                            if (coll1.length == 2 && coll2.length == 2)
+                                return Integer.parseInt(coll1[1]) - Integer.parseInt(coll2[1]);
```

Comment:
Handle `NumberFormatException` so that this doesn't blow up when `col1[1]` and `col2[1]` are not integers

Reply:
Ok ok, I will handle it, do you think it's possible that this is not a integer?

Reply:
People could use `$` at name from Alloy?

Reply:
Handled, it's returning `0` at the `catch`, is it okay for you?

Reply:
Thank you

Reply:
I don't know if it's possible, which is why suggested to catch that exception :)